src/Symfony/Component/Workflow/DataCollector/WorkflowDataCollector.php

```diff
@@ -57,4 +68,38 @@ public function getWorkflows(): array
     {
         return $this->data['workflows'] ?? [];
     }
+
+    public function getCallsCount(): int
+    {
+        $i = 0;
+        foreach ($this->getWorkflows() as $workflow) {
+            $i += \count($workflow['calls']);
+        }
+
+        return $i;
+    }
+
+    protected function getCasters(): array
+    {
+        $casters = [
+            ...parent::getCasters(),
+            TransitionBlocker::class => function ($v, array $a, Stub $s, $isNested) {
+                unset(
+                    $a[sprintf(Caster::PATTERN_PRIVATE, $v::class, 'code')],
+                    $a[sprintf(Caster::PATTERN_PRIVATE, $v::class, 'parameters')],
+                );
+
+                $s->cut += 2;
+
+                return $a;
+            },
+            Marking::class => function ($v, array $a, Stub $s, $isNested) {
+                $a[Caster::PREFIX_VIRTUAL] = array_keys($v->getPlaces());
```

Comment:
you should still name your property after the prefix.

```suggestion
                $a[Caster::PREFIX_VIRTUAL.'places'] = array_keys($v->getPlaces());
```

Reply:
rah, 🤦🏼 thanks !